### DISCUSSION TITLE
idea and request please

### DISCUSSION THREAD

**Discussion by livxbox5** [Zrips/CMI#7632] [Feature Requests]
1) you can add a scorboard module
2) Automatic addition of prefixes and suffixes to tabs (module) (without entering cmi nameplate [allPlayers] cmi nameplate [allPlayers] -pref:%cmi__user__prefix%_ -suf:_%cmi_user_suffix%

**Comment by Mrkillero4ka** [comment]
Regarding the first one, I already created an idea, you can go and vote, there I wrote in detail what it should do, maybe add something if something :)
[https://github.com/Zrips/CMI/discussions/7616](url)

**Reply by livxbox5** [reply]
The link doesn't work for you

**Reply by Mrkillero4ka** [reply]
then look in the discussions for the name:
scoreboard in cm

**Comment by livxbox5** [comment]


**Reply by Mrkillero4ka** [reply]
+

**Reply by livxbox5** [reply]
please look

**Comment by livxbox5** [comment]
idea and request

**Reply by CloudeLecaw** [reply]
would be nice to change the title to a related theme

. most posts here are ideas and requests in the discussion

**Reply by livxbox5** [reply]
I really want my idea to be brought to life

**Comment by livxbox5** [comment]
I really want my idea to be brought to life

**Reply by CloudeLecaw** [reply]
we all want that with our ideas :D  
may you could still edit your thread title be a bit nore
specialized.

**Reply by livxbox5** [reply]
for example , what name do you think is needed ?

**Reply by CloudeLecaw** [reply]
may a title like add scorebord feature

instead of a own thread we have
existing threads here were you could enjoy

https://github.com/Zrips/CMI/discussions/8152

https://github.com/Zrips/CMI/discussions/7730

https://github.com/Zrips/CMI/discussions/7616

https://github.com/Zrips/CMI/discussions/7774

adding request and ideas to an existing one is more effecve if more people discuss about, instead of each user makes it own and noone discuss.

**Comment by livxbox5** [comment]
Is it possible to rely on my request at number #7632? Or can we forget about this idea?

**Reply by CloudeLecaw** [reply]
be patient, some requests exist since over two years....
when someome wants to rply about it. then they will reply.
just be patient :-)

**Reply by livxbox5** [reply]
ok

**Comment by Angelius782** [comment]
yes it would be wonderful ... since simplescore is over and not maintain....... :(

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions/new/choose
